src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift

```diff
@@ -270,7 +270,6 @@ enum AccessibilityIdentifiers {
 			static let dataPrivacy = "QRScanner_Info_dataPrivacy"
 		}
 
-		#if targetEnvironment(simulator)
```

Comment:
Just curious why?

Reply:
With this tests can't be run on a device anymore, and we had a unit test that we wonted to run on a device as it actually produced different results there